e2e/test/scenarios/question/native-query-drill.cy.spec.ts

```diff
@@ -0,0 +1,475 @@
+
+    it("summarize drill", () => {
+      cy.log("distinct values");
+      createNativeQuestion(ordersTableQuestionDetails, {
+        visitQuestion: true,
+        wrapId: true,
+      });
+      tableHeaderClick("QUANTITY");
+      popover().findByText("Distinct values").click();
+      cy.wait("@dataset");
+      assertTableData({
+        columns: ["Distinct values of QUANTITY"],
+        firstRows: [["5"]],
+      });
+
+      cy.log("sum");
+      visitQuestion("@questionId");
+      tableHeaderClick("QUANTITY");
+      popover().findByText("Sum").click();
+      cy.wait("@dataset");
+      assertTableData({
+        columns: ["Sum of QUANTITY"],
+        firstRows: [["38"]],
+      });
+
+      cy.log("avg");
+      visitQuestion("@questionId");
+      tableHeaderClick("QUANTITY");
+      popover().findByText("Avg").click();
+      cy.wait("@dataset");
+      assertTableData({
+        columns: ["Average of QUANTITY"],
+        firstRows: [["3.8"]],
+      });
+    });
+
+    it("summarize-column-by-time drill", () => {
+      createNativeQuestion(ordersTableQuestionDetails, { visitQuestion: true });
+      tableHeaderClick("QUANTITY");
+      popover().findByText("Sum over time").click();
+      cy.wait("@dataset");
+      assertTableData({
+        columns: ["CREATED_AT: Month", "Sum of QUANTITY"],
+        firstRows: [
+          ["May 2023", "3"],
+          ["May 2024", "3"],
+          ["September 2024", "5"],
+        ],
+      });
+    });
+
+    it("unsupported drills", () => {
+      cy.log("aggregated cell click");
+      createNativeQuestion(timeseriesLineQuestionDetails, {
+        visitQuestion: true,
+      });
+      assertQueryBuilderRowCount(10);
+      cartesianChartCircle().eq(0).click();
+      popover().within(() => {
+        cy.findByText(/See these/).should("not.exist");
+        cy.findByText(/Breakout by/).should("not.exist");
+        cy.findByText(/Automatic insights/).should("not.exist");
+      });
+
+      cy.log("legend item click");
+      createNativeQuestion(timeseriesWithCategoryLineQuestionDetails, {
+        visitQuestion: true,
+      });
+      cy.findByTestId("visualization-root").findByText("Gadget").click();
+      cy.findByRole("tooltip").should("not.exist");
+    });
+  });
+
+  describe("query builder brush filters", () => {
+    it("timeseries filter", () => {
+      createNativeQuestion(timeseriesLineQuestionDetails, {
+        visitQuestion: true,
+      });
+      assertQueryBuilderRowCount(10);
+      applyBrushFilter({ left: 200, right: 800 });
+      cy.wait("@dataset");
+      assertQueryBuilderRowCount(4);
+    });
+
+    it("numeric filter", () => {
+      createNativeQuestion(numericLineQuestionDetails, {
+        visitQuestion: true,
+      });
+      assertQueryBuilderRowCount(10);
+      applyBrushFilter({ left: 200, right: 800 });
+      cy.wait("@dataset");
+      assertQueryBuilderRowCount(5);
+    });
+
+    it("coordinates filter", () => {
+      cy.log("pin map");
```

Comment:
we have at least `should apply brush filters to the native query series selecting area range when axis is a number` in `e2e/test/scenarios/visualizations-charts/line_chart.cy.spec.js`, let's keep only one